### DISCUSSION TITLE
can only access via localhost on Windows 11?

### DISCUSSION THREAD

**Discussion by xoda0** [jesec/flood#561] [General]
Hi, sorry if this is a novice question or has been asked before, but I installed Flood.us on Windows 11. I am access it on localhost, but can’t access it even on the same computer via its IP address

I already tried adding new Windows Firewall rules to allow both in & out traffic on port 3000, and disabling the Firewall altogether. Didn’t help. Any suggestions? Thanks!

![7138C730-33F4-402A-8D35-D6DE85E1AFDC](https://user-images.githubusercontent.com/10931557/168877618-872d3455-35de-41c0-8ebf-a530361bd25d.jpeg)

**Comment by jesec** [comment]
Use `-h 0.0.0.0` argument to expose Flood.

**Reply by xoda0** [reply]
Sorry, I'm really a novice to this. Running in Windows, how do I add this argument to Flood?

**Reply by jesec** [reply]
Run a terminal and then add arguments.

**Reply by xoda0** [reply]
Thanks!
